app/models/comments-model.js

```diff
@@ -19,4 +19,14 @@ exports.addComment = (comment, articleID)=>{
     const queryStr = `INSERT INTO comments (body, author, article_id) VALUES ($1, $2, $3) RETURNING *;`
     const query = db.query(queryStr, inputs)
     return Promise.all([checkExists('articles', 'article_id', articleID), query])
+}
+
+exports.removeComment = (commentID)=>{
+    const queryStr = `DELETE FROM comments WHERE comment_id = $1 RETURNING *`
+    const query = db.query(queryStr, [commentID])
+    return Promise.all([checkExists('comments', 'comment_id', commentID), query])
```

Comment:
This is absolutely fine, and a good use of your util function. However, for this end point specifically we don't actually need to do this check. We can instead check the length of rows returned, and this will tell us whether anything has been deleted or not. If nothing has been deleted, we can assume the ID doesn't exist